.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: Build
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+env:
+    REGISTRY: ghcr.io
+
+jobs:
+
+    set-env:
```

Comment:
A bit nicpicky :) But should this be named "set-vars" because we're setting an "output variable" in this case.